app/models/feed.rb

```diff
@@ -51,7 +51,7 @@ class Feed < ApplicationRecord
   validates :episode_offset_seconds, numericality: {equal_to: 0}, allow_nil: true, if: :default?
   validates :url, http_url: true
   validates :new_feed_url, http_url: true
-  validates :enclosure_prefix, http_url: true
+  validates :enclosure_prefix, http_url: true, redirect_prefix: {max_jumps: 6}, allow_blank: true
```

Comment:
6 seems right - looking for a reference on what limit to impose - 

"Maximum 4 Prefix URLs!"
https://help.prx.org/hc/en-us/articles/30939698275611-How-to-add-and-remove-enclosure-prefix-URLS-in-Dovetail

"Libsyn’s system, therefore, will limit you to adding 3 URL prepends to your show’s file URLs."
https://help.libsynsupport.com/hc/en-us/articles/360041220231-Adding-a-URL-Prepend-or-Prefix-to-Your-Show